README.md

```diff
@@ -1141,6 +1142,22 @@ ID of the stp global config. Valid values are default.
 
 Default value: default
 
+#### ios_network_dns
+
+Configure DNS settings for network devices.
+
+#### Properties
+
+The following properties are available in the `ios_network_dns` type.
+
+##### ip_domain_lookup
+
+Data type: `Optional[Boolean]`
+
+Sets whether the Domain Name Server (DNS) lookup feature should be enabled, this feature is enabled by default on most cisco machines.
```

Comment:
I think this should just read "Sets whether the Domain Name Server (DNS) lookup feature should be enabled"

Reply:
Done